### DISCUSSION TITLE
ACEBOTT QE007 Smart Home STEAM education kit could make a for a cool prototype?

### DISCUSSION THREAD

**Discussion by Hedda** [ffenix113/zigbee_home#67] [Ideas]
Could this ”ACEBOTT QE007” Smart Home STEAM education kit (currently sold as ”made for kids”) make a for a cool prototype if paired with a new custom PCB?

https://www.cnx-software.com/2024/06/09/acebott-qe007-review-a-esp32-based-smart-home-steam-education-kit-for-8-years-old-kids/

**Comment by ffenix113** [comment]
Hello Hedda,

Maybe not as a complete replacement for their original board, but I can see how this custom board can be used with button, raindrop sensor, DHT11, blue LED, photoresistor, servos and PIR sensor (maybe even LED strip). Nice set of sensors to support for this project for sure!

Though I doubt that it would be as useful as "educational kit" for kids, as the custom board (it is based on Nordic's nRF53) would require Zigbee network to be of any use.
